shared/people/invite-friends/invite-contacts.native.tsx

```diff
@@ -104,10 +104,10 @@ const InviteContacts = () => {
         <Kb.Banner color="red">{contactsErrorMessage ?? rpcErrorMessage}</Kb.Banner>
       )}
       {successCount ? (
-        <Kb.Banner color="green">{`Success! You invited ${successCount} ${pluralize(
+        <Kb.Banner color="green" style={styles.banner}>{`Yeeha! You invited ${successCount} ${pluralize(
```

Comment:
:+1:

:sheep: I haven't seen this spelling. more familiar with "Yeehaw". but I'm just one person